Library/Formula/pius.rb

```diff
@@ -10,7 +11,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     # Replace hardcoded gpg path (WONTFIX):
     # https://sourceforge.net/p/pgpius/bugs/12/
-    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
+    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
```

Comment:
It's a special dependency that can be [satisfied](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements.rb#L69-L74) by having either `gpg` or `gpg2` in the `$PATH`.

I guess if `gpg2` becomes the default soon I'm fine with this as-is. Perhaps it's worth updating the comment above the `inreplace` to reference this discussion.